images.yaml

```diff
@@ -178,7 +178,7 @@
     tag: v2.6.2-v1.6.0
 - name: golang
   patterns:
-  - pattern: '>= 1.13.1'
+  - pattern: '>= 1.13.1-alpine3.10'
```

Comment:
Given that semver didn't realize this wasn't a prerelease, is it possible this would include future/other prereleases? Running locally with --dry-run I get 
`D 03/04 03:52:48 Found 124 upstream tags which match the pattern >= 1.13.1-alpine3.10 | retagger/pkg/retagger/job_pattern.go:52`